mobile/components/Loading/index.js

```diff
@@ -0,0 +1,23 @@
+// @flow
+import * as React from 'react';
+import { ActivityIndicator } from 'react-native';
+import { Container } from './style';
+import theme from '../theme';
```

Comment:
Instead of importing the theme here, use `withTheme` from styled-components which injects the dynamic theme from the context to the props:

```JS
class Loading extends React.Component {
  render() {
    const { theme } = this.props;
    return ()
  }
}

export default compose(withTheme)(Loading);
```

That way, if we decide to add a night mode or something it'll work out of the box.